myia/operations/prim_random_uniform.py

```diff
@@ -0,0 +1,58 @@
+"""Definitions for the primitive `random_uniform`."""
+
+from .. import xtype
+from ..lib import (
+    ANYTHING,
+    SHAPE,
+    TYPE,
+    VALUE,
+    AbstractArray,
+    AbstractRandomState,
+    AbstractScalar,
+    AbstractTuple,
+    standard_prim,
+)
+from . import primitives as P
+
+
+def pyimpl_random_uniform(rstate, low, high, size):
+    """Implement `random_uniform`."""
+    value = rstate.uniform(low, high, size)
+    return rstate, value
+
+
+@standard_prim(P.random_uniform)
+async def infer_random_uniform(self, engine,
+                               rstate: AbstractRandomState,
+                               low: AbstractScalar,
+                               high: AbstractScalar,
+                               size: AbstractTuple):
```

Comment:
I think we should standardize on the name `shape` for shape arguments.

Reply:
Ok!